plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/LineRange.java

```diff
@@ -0,0 +1,16 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class LineRange {
```

Comment:
I think this class needs `equals` and `hashCode` methods to be able to sort & compare them.